### PR TITLE
Google Consent Mode [DOC-839]

### PR DESCRIPTION
**Pull request by stayseesong**

### CODE REVIEW

File: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md [hunk 1]
```diff
@@ -65,7 +62,72 @@ The automatically collected and enhanced measurement events include parameters b
...
+            * Ad User Data Consent State
+            * Ad Personalization Consent State
+            You can manually select `Granted` or `Denied` from the dropdown menu for Advanced consent mode settings, and type in `granted` or `denied` for basic consent mode settings. 
+  
+      * **Option 2:** 
+        Create an event variable to directly grab the value from the payload. To do this: 
+
+        1. Navigate to **Connections > Destinations** and select your Google Analytics 4 Web destination. 
+        2. Go to the **mappings** tab of the destination. 
+        3. Select the mapping you want to edit.'
```
Comment (rvadera12) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/index.md:120]: Maybe we can combine option 1 and 2, only step 4 is different

File: src/connections/destinations/catalog/actions-google-analytics-4-web/index.md [hunk 2]
```diff
@@ -65,7 +62,72 @@ The automatically collected and enhanced measurement events include parameters b
...
+        4. Under the **Select mappings** section, create an event variable to directly grab the value from the payload. Ensure it translates to `granted` or `denied`. You can use an insert function to translate to `granted` or `denied`. Use the replace function if it's a string. Do this for these fields:
```
Comment (rvadera12) [inline src/connections/destinations/catalog/actions-google-analytics-4-web/index.md:121]:
```suggestion
        4. Under the **Select mappings** section, create an event variable to directly grab the value from the payload (ex: `properties.adStorageConsentState`). Ensure it translates to `granted` or `denied`. You can use an insert function to translate to `granted` or `denied`. Use the replace function if it's a string. Do this for these fields:
```

**Review by rvadera12 — COMMENTED** [review]
Looks great! Left a couple of comments. Could we also replace the info banner with, Google is enforcing consent starting March 6th and here are instructions to implement this with a link in the docs to the consent section. We can plan to remove this in the future, but just so it is easier for customers to find it

File: src/connections/destinations/catalog/google-ads-classic/index.md [hunk 3]
```diff
@@ -26,6 +26,27 @@ You can also trigger Google Ads (Classic) conversion from your mobile app using
...
+
+Consent mode in the Gtag library and Google Analytics is designed to help website owners comply with privacy regulations, such as the General Data Protection Regulation (GDPR) in the European Union. It allows website owners to adjust how these tools use and collect data based on user consent.
+
+With consent mode, you can configure your website to dynamically adjust the tracking behavior of the Gtag library and Google Analytics based on the user's consent status. If a user provides consent to data processing, both the Gtag library and Google Analytics can collect and use that data for analysis. If a user doesn't provide consent, both tools limit data collection to essential functions, helping businesses respect user privacy preferences.
+
+Consent mode may involve updates to your sources outside of Segment, such as incorporating a consent management system for consent functionality.
+
+To set up consent mode for Google Ads Classic:
```
[the remark this answers is not in the document]
Comment (stayseesong) [reply src/connections/destinations/catalog/google-ads-classic/index.md:38]: @rvadera12 why did you add "App Stream" when the destination is called Google Ads Classic?

File: src/connections/destinations/catalog/google-ads-classic/index.md [hunk 4]
```diff
@@ -26,6 +26,36 @@ You can also trigger Google Ads (Classic) conversion from your mobile app using
...
+
+With consent mode, you can configure your website to dynamically adjust the tracking behavior of the Gtag library and Google Analytics based on the user's consent status. If a user provides consent to data processing, both the Gtag library and Google Analytics can collect and use that data for analysis. If a user doesn't provide consent, both tools limit data collection to essential functions, helping businesses respect user privacy preferences.
+
+Consent mode may involve updates to your sources outside of Segment, such as incorporating a consent management system for consent functionality.
+
+### Consent mode for mobile and server apps
+To set up consent mode for Google Ads Classic mobile and server apps:
+1. Update your app's SDK to a version that supports consent mode v2. 
+  * Android apps must use F[irebase Android Analytics SDK version 21.5.0 or later](https://firebase.google.com/support/release-notes/android#analytics_v21-5-0){:target="_blank"}.
```
Comment (pwseg) [inline src/connections/destinations/catalog/google-ads-classic/index.md:41]:
```suggestion
  * Android apps must use [Firebase Android Analytics SDK version 21.5.0 or later](https://firebase.google.com/support/release-notes/android#analytics_v21-5-0){:target="_blank"}.
```

**Review by pwseg — APPROVED** [review]

**Review by rvadera12 — APPROVED** [review]

**Pull request merged**